resources/icons/16px/inherited-group

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file is missing the `.svg` file extension, which is expected by the [compile script](https://app.circleci.com/pipelines/github/palantir/blueprint/63/workflows/880eb031-648c-4c20-aaca-ca5a22dcc88a/jobs/31607/parallel-runs/0/steps/0-104):

```
lerna ERR! yarn run compile stderr:
 
(node:254) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/circleci/project/resources/icons/16px/inherited-group.svg'
 
```